File: app/src/main/java/com/dgd/earthquakes/data/network/callback/IQuakesCallbackListener.java
package com.dgd.earthquakes.data.network.callback;

import com.dgd.earthquakes.data.network.infra.QuakeData;

import java.util.List;

/**
 * Created by Max
 * on 30-Apr-17.
 */
public interface IQuakesCallbackListener {
    void onNetworkError(String errorMessage, int errorCode);
    void onNetworkSuccess(List<QuakeData> quakes);
}

File: app/src/main/java/com/dgd/earthquakes/data/network/callback/IQuakesCallbackListener.kt
package com.dgd.earthquakes.data.network.callback

import com.dgd.earthquakes.data.network.infra.QuakeData

/**
 * Created by Max
 * on 30-Apr-17.
 */
interface IQuakesCallbackListener {
    fun onNetworkError(errorMessage: String, errorCode: Int)
    fun onNetworkSuccess(quakes: List<QuakeData>)
}
